src/youwol/app/environment/proxied_esm_servers.py

```diff
@@ -0,0 +1,242 @@
+# standard library
```

Comment:
Maybe this should be corrected; `pyupgrade` can be run without installation with `pipx`:
`$ pipx run pyupgrade …`

Reply:
I missed it, thanks.